modules/python/python-logmsg.c

```diff
@@ -82,8 +82,9 @@ _py_log_message_subscript(PyObject *o, PyObject *key)
 
   PyLogMessage *py_msg = (PyLogMessage *) o;
   PyObject *value = _get_value(py_msg, name, py_msg->cast_to_bytes);
-  if (!value)
+  if (value == Py_None)
```

Comment:
Same as my previous comment: `Py_None` is a valid value for `LM_VT_NULL`. I think we should only throw a warning if `value` is `NULL`, which means we do not have an NV pair with such a `name` or we have some other failure.

Reply:
Throw an exception*, sorry.

Reply:
Fixed. I had to use `log_msg_get_value_if_set_with_type()`, because `log_msg_get_value_with_type` returns a NULL-type for unset values.